config/ember-try.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-env node */
 module.exports = {
+  useYarn: true,
```

Comment:
@alexdiliberto Curious if this update addresses https://github.com/ember-cli/ember-try/issues/148?

Reply:
@crodriguez1a Yea this specifically fixes that 2.12 CI issue